lib/router/config-router.js

```diff
@@ -104,13 +104,8 @@ function ConfigRouter(conf, schedulerRegistry) {
   this.servers = createServers(conf);
   this.accessLogSpec = createAccessLogSpec(conf.getOne('access_log'));
   this.socketDir = conf.getValues('socket_dir').path;
-  
-  this.disabledProtocols = conf.getValues('disable_protocols').names || [];
-  if (conf.getValues('disable_websockets').val){
-    this.disabledProtocols.push('websocket');
-  }
 
-  this.reconnect = false;
+  this.reconnect = true;
```

Comment:
Try to remove this block by unifying the createServer sockjs prefix for both reconnect and non-reconnect cases.